books/free-programming-books-id.md

```diff
@@ -55,6 +55,7 @@
 ### Git
 
 * [Belajar Git untuk Pemula](https://github.com/petanikode/belajar-git) - Ahmad Muhardian, Petani Kode (HTML)
+* [Dasar Pemrograma Golang](https://santrikoding.com/ebook/dasar-pemrograman-golang) - Ebook dari Santri Koding
```

Comment:
```suggestion
* [Dasar Pemrograma Golang](https://santrikoding.com/ebook/dasar-pemrograman-golang) - Noval Agung Prayogo
```

Reply:
am i wrong?

Reply:
yep, you're right. Sorry, I forgot to put it in the GO section and there was a typo. But I've fixed it.

Thanks.